scripts/installer.py

```diff
@@ -192,4 +199,10 @@ def startInstall(self):
             self.generateCACertificate()
         else:
             logging.info("CA certificate already installed.")
-        logging.info("Thyra installed successfully")
+
+        logging.info("Thyra installed successfully !")
+        logging.info(f"Thyra has been installed in {self.THYRA_INSTALL_FOLDER_PATH}")
```

Comment:
From the QA needs: "Check that it is installing thyra-server and thyra-app in the directory displayed at the end of the script."

```suggestion
        logging.info(f"thyra-server and thyra-app have been installed in {self.THYRA_INSTALL_FOLDER_PATH}")
```

Furthermore, the binary name of thyra on macos is thyra_server